CHANGELOG.md

```diff
@@ -16,6 +16,8 @@
 ### Enhancements & Refactors
 - [#1108](https://github.com/openscope/openscope/issues/1108) - Overhaul of KLAS
 - [#794](https://github.com/openscope/openscope/issues/794) - Bring KMIA back into compliance with the airport specs
+- [#1114](https://github.com/openscope/openscope/issues/1114) - Update broken link to "Airport documentation" in README.md
```

Comment:
I think #1115 covers both, no need for two entries

Reply:
I thought one entry for each resolved issue but #1114 can be closed actually.

Reply:
Yep it has been. So #1115 only.